pom.xml

```diff
@@ -256,6 +275,54 @@
                 <version>3.9</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.swagger.codegen.v3</groupId>
+                <artifactId>swagger-codegen-generators</artifactId>
+                <version>1.0.19</version>
+            </dependency>
+            <dependency>
+                <groupId>io.swagger</groupId>
+                <artifactId>swagger-core</artifactId>
+                <version>1.6.1</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-ext</artifactId>
+                <version>1.7.30</version>
+            </dependency>
```

Comment:
Is this supplied by the BR parent POM? I worry about version conflicts.

Reply:
Good concern :+1: 

Short answer is: it's not - it's a transitive dependency, first introduced by the `content-exim-core` module. Swagger CodeGen library introduces it as a transitive dependency as well, but with a slightly different version.

That version inconsistency was flagged by `maven-enforcer-plugin` which showed all the (different) transitively included versions and what requires them. I specified this dependency in the `dependencyManagement` section of the POM file to explicitly 'fix' it at the highest version of the 1.7.x line that was mentioned in the Maven Enforcer's report.

Luckily, as shown by the report below, all versions of the those dependencies differ only in their 'patch' segment so I don't expect any incompatibilities:

```
[WARNING] Rule 2: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.slf4j:slf4j-ext:1.7.12 paths to dependency are:
+-uk.nhs.digital:website-cms:dev-SNAPSHOT
  +-io.swagger.codegen.v3:swagger-codegen-generators:1.0.19
    +-org.slf4j:slf4j-ext:1.7.12
and
+-uk.nhs.digital:website-cms:dev-SNAPSHOT
  +-org.onehippo.forge.content-exim:content-exim-core:3.0.1
    +-org.slf4j:slf4j-ext:1.7.25
and
+-uk.nhs.digital:website-cms:dev-SNAPSHOT
  +-io.swagger.codegen.v3:swagger-codegen-generators:1.0.19
    +-io.swagger.codegen.v3:swagger-codegen:3.0.19
      +-org.slf4j:slf4j-ext:1.7.25
and
+-uk.nhs.digital:website-cms:dev-SNAPSHOT
  +-org.onehippo.cms7:hippo-cms-test:13.4.2
    +-org.onehippo.cms7:hippo-repository-testutils:13.4.2
      +-org.slf4j:slf4j-ext:1.7.25
and
+-uk.nhs.digital:website-cms:dev-SNAPSHOT
  +-io.swagger.codegen.v3:swagger-codegen-generators:1.0.19
    +-io.swagger.codegen.v3:swagger-codegen:3.0.19
      +-io.swagger:swagger-parser:1.0.50
        +-org.slf4j:slf4j-ext:1.7.28
and
+-uk.nhs.digital:website-cms:dev-SNAPSHOT
  +-io.swagger.codegen.v3:swagger-codegen-generators:1.0.19
    +-io.swagger.codegen.v3:swagger-codegen:3.0.19
      +-io.swagger.parser.v3:swagger-parser:2.0.19
        +-org.slf4j:slf4j-ext:1.7.30
and
+-uk.nhs.digital:website-cms:dev-SNAPSHOT
  +-io.swagger.codegen.v3:swagger-codegen-generators:1.0.19
    +-io.swagger.codegen.v3:swagger-codegen:3.0.19
      +-io.swagger:swagger-codegen:2.4.13
        +-org.slf4j:slf4j-ext:1.7.12
```